monitoring/angle-monitoring/src/main/java/com/farao_community/farao/monitoring/angle_monitoring/AngleMonitoring.java

```diff
@@ -221,8 +221,18 @@ private AngleMonitoringResult monitorAngleCnecs(String loadFlowProvider, LoadFlo
     private Map<AngleCnec, Double> computeAngles(Set<AngleCnec> angleCnecs, Network network) {
         Map<AngleCnec, Double> anglePerCnec = new HashMap<>();
         angleCnecs.forEach(ac -> {
-            VoltageLevel exportingVoltageLevel = network.getVoltageLevel(ac.getExportingNetworkElement().getId());
-            VoltageLevel importingVoltageLevel = network.getVoltageLevel(ac.getImportingNetworkElement().getId());
+            VoltageLevel exportingVoltageLevel;
```

Comment:
- create a function to avoid dubplication.
- use network.getIndentifiable() to get the element in the network; verify that it is not null, then use Identifiable::getType to know if it's a bus or a voltage level
- add unit tests

Reply:
It seem that a bus doesn't really have an id (at least for the network used in the unit test class, network.xiidm), so network.getIdentifiable return null with the bus id (derived from the VoltageLevel id), while it is still accessible from the busBreakerView, hence the ordering of the if in the function I have added.